src/docs/product/crons/getting-started/cli.mdx

```diff
@@ -59,6 +59,32 @@ sentry-cli monitors run my-monitor-slug -- python path/to/file.py
 sentry-cli monitors run my-monitor-slug -- node path/to/file.js
 ```
 
+### Configuring Your Monitor via the CLI
+
+You can also use the Sentry CLI to configure your cron monitor when you run your job. This way, you can avoid having to first set up the monitor through the Sentry web interface.
```

Comment:
This line feels weird in the middle of the page since it's general to the CLI and not specific to this section. I think you can either delete it or move it up to right before the "Requirements" section.

Reply:
Can you please explain why you think this line is out of place here? I think it is necessary to include this line here to introduce the purpose of this feature; that is, that you can configure your monitor via the CLI instead of configuring via the web interface. The line also indicates that using this feature is optional, since monitors can also be created and configured in the Sentry web interface.

However, I agree that we should probably mention the possibility of setting up the monitor via the CLI itself in the "Requirements" section, so I will add a mention there.

Reply:
Ah I thought it was implied from where the page lives (Crons -> Set Up -> CLI) that you were using the CLI to create / set up a new monitor, so it felt like something that should go at the top of the page. But now I see that the original feature was just for checking in on an existing monitor.

I think to make this clearer, you could keep this line here, but rename the section header to "Create or Update Your Monitor" similar to the section for [HTTP](https://docs.sentry.io/product/crons/getting-started/http/#creating-or-updating-a-monitor-through-a-check-in-optional).